integration_tests/tests/sql/test_get_filtered_columns_in_relation.sql

```diff
@@ -0,0 +1,5 @@
+{% set actual_output = dbt_utils.get_filtered_columns_in_relation(from= ref('data_star_uppercase_columns'), except=['field_1'], output_lower=True) %}
+
+{% set expected_output = ['field_2', 'field_3'] %}
+
+{{ assert_equal_values (actual_output | trim, expected_output | trim) }}
```

Comment:
This is cool 🔥  out of curiosity, is object equivalency in Jinja/Python order-dependent? i.e. would `['field_3', 'field_2']` pass? I don't have strong feelings about whether it should or not (if you pushed me on it, I'd say they're _not_ equal), but feels important to know.